extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/stream/YoutubeStreamExtractorDefaultTest.java

```diff
@@ -59,54 +60,54 @@
     public static final String YOUTUBE_LICENCE = "YouTube licence";
 
     public static class NotAvailable {
-        @BeforeClass
+        @BeforeAll
         public static void setUp() throws IOException {
             YoutubeParsingHelper.resetClientVersionAndKey();
             YoutubeParsingHelper.setNumberGenerator(new Random(1));
             YoutubeStreamExtractor.resetDeobfuscationCode();
             NewPipe.init(new DownloaderFactory().getDownloader(RESOURCE_PATH + "notAvailable"));
         }
 
-        @Test(expected = GeographicRestrictionException.class)
-        public void geoRestrictedContent() throws Exception {
+        @Test
+        void geoRestrictedContent() throws Exception {
             final StreamExtractor extractor =
                     YouTube.getStreamExtractor(BASE_URL + "_PL2HJKxnOM");
-            extractor.fetchPage();
+            assertThrows(GeographicRestrictionException.class, extractor::fetchPage);
         }
 
-        @Test(expected = ContentNotAvailableException.class)
-        public void nonExistentFetch() throws Exception {
+        @Test
+        void nonExistentFetch() throws Exception {
             final StreamExtractor extractor =
                     YouTube.getStreamExtractor(BASE_URL + "don-t-exist");
-            extractor.fetchPage();
+            assertThrows(ContentNotAvailableException.class, extractor::fetchPage);
         }
 
-        @Test(expected = ParsingException.class)
-        public void invalidId() throws Exception {
+        @Test
+        void invalidId() throws Exception {
             final StreamExtractor extractor =
                     YouTube.getStreamExtractor(BASE_URL + "INVALID_ID_INVALID_ID");
-            extractor.fetchPage();
+            assertThrows(ParsingException.class, extractor::fetchPage);
         }
 
-        @Test(expected = PaidContentException.class)
-        public void paidContent() throws Exception {
+        @Test
+        void paidContent() throws Exception {
             final StreamExtractor extractor =
                     YouTube.getStreamExtractor(BASE_URL + "ayI2iBwGdxw");
-            extractor.fetchPage();
+            assertThrows(PaidContentException.class, extractor::fetchPage);
         }
 
-        @Test(expected = PrivateContentException.class)
-        public void privateContent() throws Exception {
+        @Test
+        void privateContent() throws Exception {
             final StreamExtractor extractor =
                     YouTube.getStreamExtractor(BASE_URL + "8VajtrESJzA");
-            extractor.fetchPage();
+            assertThrows(PrivateContentException.class, extractor::fetchPage);
         }
 
-        @Test(expected = YoutubeMusicPremiumContentException.class)
-        public void youtubeMusicPremiumContent() throws Exception {
+        @Test
+        void youtubeMusicPremiumContent() throws Exception {
```

Comment:
Why did you remove `public`?

Reply:
https://rules.sonarsource.com/java/RSPEC-5786